src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -1260,9 +1261,20 @@ public Page<BedAssignment> getBedAssignmentsHistory(@PathVariable("bookingId") @
         @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
         @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
-    @Operation(summary = "Gets the return to custody date for a booking")
+    @Operation(summary = "Gets the return to custody date for a booking", description = "Do not use - replaced with /fixed-term-recalls", deprecated = true, hidden = true)
     @GetMapping("/{bookingId}/return-to-custody")
     public ReturnToCustodyDate getReturnToCustodyDate(@PathVariable("bookingId") @Parameter(description = "The offender booking linked to the return to custody date.", required = true) final Long bookingId) {
```

Comment:
This endpoint and associated downstream code shall be removed  after calculate-release-dates has been changed to use the new endpoint